tools/bmh_test/run_local_bmh_test_setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Check if the script is run as root
+if [[ $UID != 0 ]]; then
```

Comment:
```suggestion
if [[ ${EUID} != 0 ]]; then
```
You should be checking effective UID, `${EUID}` as that cannot be faked.